NEWS.md

```diff
@@ -176,7 +178,7 @@
   outcome for the next best dose among admissible doses, thus avoiding early stopping
   at the placebo dose level.
   
-* If DataMixture objects are used, mcmc now correctly sets fromPrior to FALSE if
+* If DataMixture objects are used, mcmc now correctly sets from_prior to FALSE if
```

Comment:
that is in the past, I would not change this

Reply:
Sure, but then I have to add it into `WORDLIST` and actually this `fromPrior` is actually not used by `mcmc`.